src/wallet/apiHelpers.ts

```diff
@@ -1,31 +1,57 @@
 import { Api } from 'custom-types';
 import { API_ENDPOINT } from '../../config.json';
 
+class ApiError extends Error {
+  constructor(...args: any[]) {
+    super(...args);
+    this.name = 'ApiError';
+    Error.captureStackTrace(this, ApiError);
+  }
+}
+
+// TODO: handle pagination
```

Comment:
Is this still a TODO here? Fine to leave it as such, just curious about the status. Maybe we can create an issue for it?